posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -0,0 +1,389 @@
+---
+layout: post
+title: "Microprofile Long Running Actions in Open Liberty"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yongja79
+author_github: https://github.com/yongja79
+seo-title: Microprofile Long Running Actions in Open Liberty
+seo-description: Try Microprofile Long Running Actions in the Open Liberty 20.0.0.12-beta
+blog_description: "Try Microprofile Long Running Actions in the Open Liberty 20.0.0.12-beta"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Microprofile Long Running Actions in Open Liberty
+Jason Yong <https://github.com/yongja79>
+
+In the 20.0.0.12-beta release of Open Liberty a new capability was introduced to support loosely-coupled transaction semantics for Java microservice. This is an implementation of a new MicroProfile specification for Long Running Actions (LRAs).  
+
+== What is a LRA?
```

Comment:
Can you add a section on why you need MP LRA? Why is 2-phase commit not suitable?

Reply:
The why is actually in the "So how does MicroProfile LRA help?" section which had a comparison with 2PC in an earlier draft but was taken out by Ian R as he didn't feel it needed it.